src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UsePropertyInsteadOfCountMethodWhenAvailableTests.cs

```diff
@@ -17,6 +17,46 @@ namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
 {
     public static partial class UsePropertyInsteadOfCountMethodWhenAvailableTests
     {
+        [Fact]
+        public static Task CSharp_AsMethodArgument_Tests()
+            => new VerifyCS.Test
+            {
+                TestState =
+                {
+                    Sources=
+                    {
+                $@"using System;
+using System.Linq;
+public static class C
+{{
+    public static System.Collections.Immutable.ImmutableArray<int> GetData() => default;
+    public static void M()
+    {{
+        var b = 1.Equals({{|{UseCountProperlyAnalyzer.CA1829}:GetData().Count()|}});
```

Comment:
Can you add tests where you have parenthesis wrapping `GetData().Count()` and in between?
* `(GetData().Count())`
* `(GetData()).Count()`

And the same for VB.